gpkit/interactive/sensitivity_map.py

```diff
@@ -29,6 +29,7 @@ def colorfn(senss):
 def signomial_print(sig, sol, colorfn, paintby="constants", idx=None):
     "For pretty printing with Sympy"
     mstrs = []
+    color_list = []
```

Comment:
So the general process I used was to use a list to get the colors from signomial print, then add them to a color list in the sensitivity map code, then sort that list so the color bar is sorted. This is why I used a list. However, I could make this first color list a set, and then use something like this to add those components to a list that can be sorted later if you prefer. However, the colors need to be in a list eventually so they can be sorted.
``` python
for i in range(len(p)):
    print p.pop()
```